test/export/test_export.py

```diff
@@ -3854,6 +3858,22 @@ def foo_unbacked_fake_impl(a, b):
         self.assertEqual(type(path[2]).__name__, "DivideByKey")
         self.assertEqual(path[2].divisor, 10)
 
+        # collect bound symbols
+        bound = set()
+        for node in ep.graph.nodes:
+            bound.update(node.meta.get("unbacked_bindings", {}))
+
+        # check ShapeEnv counters compared to binding indices
+        shape_env = detect_fake_mode(
+            [node.meta.get("val") for node in ep.graph.nodes]
+        ).shape_env
+        next_index = next(shape_env.unbacked_symint_counter)
+        for symbol in bound:
```

Comment:
Curious to see what happens when we have a `torch._check` or some kind of assert that makes the unbacked symbol go to a constant or backed symint. The test I suggested should have some of that...

Reply:
Oh maybe the next test `test_replaced_unbacked_bindings` is for that?